lib/runner.js

```diff
@@ -124,6 +124,11 @@ exports.execute = function (scripts, options, reporter, callback) {
         return script._root;
     });
 
+    const onlyNodes = scripts.reduce((nodes, script) => nodes.concat(script._onlyNodes), []);
```

Comment:
Isn't it more readable with a `Hoek.flatten(scripts.map((script) => script._onlyNodes))` ? Personal tastes...